tools/vagrant/README.md

```diff
@@ -176,46 +186,46 @@ cd ${OPENWHISK_HOME}
 ./gradlew distDocker
 ```
 
-## Safe Re-deploy (after VM restart)
+## Using docker-runc
+Only for experimental use:
+To use docker-runc the docker-runc CLI use in the invoker needs to match the version on the docker engine host.
+Get the version of the docker engine like the following:
+```
+$ docker version | grep Version
+Version:	18.03.0-ce
+```
+Update the Invoker Dockerfile to download a newer version of the docker-cli
+https://download.docker.com/linux/static/stable/x86_64/docker-18.03.0-ce.tgz
 
-If you restart the VM (e.g., `vagrant reload`), it may be necessary to refresh
-the OpenWhisk deployment. You can do this in a way that does not reload the data
-store container.
+Edit the [core/invoker/Dockefile](../../core/invoker/Dockefile)
+Update the variable with the version
+```
+ENV DOCKER_VERSION 18.03.0-ce
+```
+Then update line with the curl download command like
+```
+RUN curl -sSL -o docker-${DOCKER_VERSION}.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz && \
+```
+Then update the ansible configuration to enable the use of runc, edit [](../../ansible/environments/vagrant/group_vars/all)
+```
+invoker_use_runc: true
```

Comment:
will the following Vagrantfile command update in this PR
```su vagrant -c 'ansible-playbook -i environments/vagrant openwhisk.yml -e invoker_use_runc=False'```
override that ansible property value?

Reply:
yes